packages/office-ui-fabric-react/src/components/Coachmark/PositioningContainer/PositioningContainer.styles.ts

```diff
@@ -21,41 +20,22 @@ export interface IPositioningContainerNames {
   beakCurtain?: string;
 }
 
-// @TODO Remove this tslint disable statement after the styles are converted
-// to the updated mergeStyles method.
-/* tslint:disable */
-export function highContrastActive(styles: IStyle): any {
-  return {
-    '@media screen and (-ms-high-contrast: active)': styles
-  };
-}
-
-export function focusClear(): any {
-  return {
-    '&::-moz-focus-inner': {
-      border: 0
-    },
-    '&': {
-      outline: 'transparent'
-    }
-  };
-}
-/* tslint:enable */
-
 export const getClassNames = memoizeFunction(
   (): IPositioningContainerNames => {
     return mergeStyleSets({
-      root: {
-        position: 'absolute',
-        boxSizing: 'border-box',
-        border: '1px solid ${}',
-        selectors: {
-          ...highContrastActive({
-            border: '1px solid WindowText'
-          }),
-          ...focusClear()
-        }
-      },
+      root: [
+        {
+          position: 'absolute',
+          boxSizing: 'border-box',
+          border: '1px solid ${}',
+          selectors: {
+            [HighContrastSelector]: {
+              border: '1px solid WindowText'
+            }
+          }
+        },
+        focusClear()
```

Comment:
[focusClear definition](https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/styling/src/styles/getFocusStyle.ts) for reference (it's functionally identical to what the custom method was doing)